src/api/track.ts

```diff
@@ -160,15 +160,41 @@ class TrackAPI {
           sclVal,
           cslVal,
         };
+
+        let { user, events, token, expiresAt } = response;
+        const location = { latitude, longitude, accuracy };
+        let passed;
+        let expiresIn;
+        if (expiresAt) {
+          expiresAt = new Date(expiresAt);
+          passed = user?.fraud?.passed && user?.country?.passed && user?.state?.passed;
+          expiresIn = (expiresAt.getTime() - Date.now()) / 1000;
```

Comment:
should the name be `expiresInSeconds` to be explicit? No strong opinion here because jwt uses `expiresIn` as well and just has the docs saying it is in seconds

Reply:
We don't really include units in the field name anywhere else in our APIs, so prefer to keep it `expiresIn` (if I could go back in a time machine to 2017 I might have used this pattern, but too late now, rather optimize for consistency and brevity)